code/game/objects/items/weapons/tools/hammer.dm

```diff
@@ -112,13 +112,14 @@
 
 /obj/item/tool/hammer/ironhammer
 	name = "GmbH \"Ironhammer\" Breaching Hammer"
-	desc = "A modified sledgehammer produced by Seinmetall for police forces across Sol space. This tool can take down standard walls and if the user is strong enough, reinforced walls."
+	desc = "A modified sledgehammer produced by Seinmetall for police forces across Sol space.  "
```

Comment:
```suggestion
	desc = "A modified sledgehammer produced by Seinmetall for police forces across Sol space."
```

Reply:
why? It was buffed explicitly to be able to take down r-walls, hence the addition of delay.

Reply:
You're the one who removed the r-wall part.
Trilby's asking to remove a superfluous space.

Reply:
I'm dumb as fuck and gay too